kani-driver/src/main.rs

```diff
@@ -125,6 +136,15 @@ fn standalone_main() -> Result<()> {
         let result = ctx.check_harness(&specialized_obj, &report_dir, harness)?;
         if result == VerificationStatus::Failure {
             failed_harnesses.push(harness);
+
+            if have_seen_failed_harness && ctx.args.add_exe_trace_to_src && !ctx.args.quiet {
```

Comment:
Agreed, it would mess up user code if we tried to insert more than one det unit tests (primarily because the line numbers would be messed up). I plan to fix this in the future. I added it to the issue.

Done. Used a static variable in the parser.

Reply:
TODO: Require harness is present

Reply:
Done